scalafmt-tests/src/test/resources/scala3/OptionalBraces.stat

```diff
@@ -629,3 +629,269 @@ object a:
      // c1
      else if aa then aaa
 // c1
+<<< rewrite with empty blocks
+rewrite.scala3.useOptionalBraces = true
+===
+object a {
+  def a = {
+  }
+}
+>>>
+object a:
+   def a = {}
+<<< rewrite with nested if-else
+rewrite.scala3.useOptionalBraces = true
+===
+object a {
+  val a = {
+    if (a) {
+      if (aa) {
+        aaa
+      } // c1
+    } else {
+      b
+    }
+  }
+}
+>>>
+object a:
+   val a =
+     if (a) {
```

Comment:
probably, but it's not a scala3 rewrite with optional braces and significant indentation, because that requires a `then`, if I'm reading the doc correctly.

this is what our regular redundant braces rule would do.